tests/unit_tests/test_tokenizer.py

```diff
@@ -23,7 +23,7 @@
 
 
 class TestTokenizerIntegration(unittest.TestCase):
```

Comment:
nit: in a follow up PR you could also add unit tests to validate the assets used in state dict are also downloaded and used correctly?